tests/Organization.py

```diff
@@ -324,13 +326,18 @@ def testCreateRepoWithAllArguments(self):
             allow_merge_commit=False,
             allow_rebase_merge=True,
             delete_branch_on_merge=False,
+            visibility="internal",
         )
         self.assertEqual(
             repo.url, "https://api.github.com/repos/BeaverSoftware/TestPyGithub2"
         )
         self.assertFalse(repo.has_wiki)
         self.assertFalse(repo.has_pages)
 
+    def testCreateRepoWithInvalidVisibility(self):
+        with pytest.raises(AssertionError):
```

Comment:
``with self.assertRaises()`` instead of importing pytest.

Reply:
Perfect. Thanks!